packages/web/src/components/toolbarButton.css

```diff
@@ -16,15 +16,25 @@
 
 .toolbar-button {
   flex: none;
-  border: none;
-  outline: none;
   color: var(--vscode-sideBarTitle-foreground);
   background: transparent;
-  padding: 4px;
   margin-left: 10px;
-  cursor: pointer;
   display: inline-flex;
   align-items: center;
+  border-radius: 6px;
+  border: 1px solid var(--vscode-dropdown-border);
```

Comment:
ditto

Reply:
these seems to be best color for border for both light and dark mode as inspired by github buttons